data.csv

```diff
@@ -5,3 +5,4 @@ Paige Pierce,90,59:59,59:54,59:40,59:28,59:09,58:48,58:38,57:45,52:18,38:43,13:2
 Luke Callaghan (DGK),90,59:59,59:50,59:45,58:58,53:41,52:51,52:32,50:43,38:25,23:26,6:17,0:00,,,,,,,,
 Bradley Williams 2.0,80,59:50,59:34,59:26,58:59,57:59,57:46,55:35,51:25,41:25,29:57,22:20,20:13,12:50,2:05,1:08,,,,,
 Disc Golf with Brady,80,59:58,59:49,59:39,59:26,58:45,58:12,53:42,50:44,49:06,47:31,45:33,25:41,19:21,14:08,11:48,10:58,6:57,,,
+Andrew Preznell,110,59:59,59:49,58:23,57:00,55:43,53:47,51:42,44:07,31:55,31:35,27:17,16:31,5:35
```

Comment:
```suggestion
Andrew Preznell,70,59:59,59:49,58:23,57:00,55:43,53:47,51:42,44:07,31:32,27:14,16:31,5:26
```

thanks for compiling this!  Definitely one of the hardest videos to transcribe..  I got some different data for some of the throws + I believe his final make is at 120ft (not 130ft)